docs/changelog.md

```diff
@@ -0,0 +1,52 @@
+### 24-July-2020 - 12:52 CEST
+-  [fix] Renamed Jenkins project from conan-center-pull-request to cci to improve issues with long workspace paths in Windows agents.
+-  [feature] Updated Conan client to the 1.27.1 version in Windows and Mac agents.
```

Comment:
Sorry, I forgot to update the time. It is important in order to track changes. I have updated the wiki as well

```suggestion
### 24-July-2020 - 13:05 CEST
- Renamed Jenkins project from `conan-center-pull-request` to `cci` to improve issues with long workspace paths in Windows agents.

### 24-July-2020 - 12:52 CEST
- Updated Conan client to the 1.27.1 version in Windows and Mac agents.
```